Modules/DataModel/Sources/PocketCastsDataModel/Private/Managers/EndOfYearDataManager.swift

```diff
@@ -256,6 +256,32 @@ class EndOfYearDataManager {
         return episode
     }
 
+    /// Given a list of UUIDs, return which UUIDs are present on the database
+    func episodesThatExists(dbQueue: FMDatabaseQueue, uuids: [String]) -> [String] {
+        var episodes: [String] = []
+
+        dbQueue.inDatabase { db in
+            do {
+                let query = """
+                            SELECT uuid FROM \(DataManager.episodeTableName) WHERE \(DataManager.episodeTableName).uuid IN \(DBUtils.valuesQuestionMarks(amount: uuids.count)) and
```

Comment:
When I ran the query on my local DB I ended up with multiple results for the same episode (not sure how). I think we can limit the chance of that happening by doing:

```suggestion
                            SELECT DISTINCT uuid FROM \(DataManager.episodeTableName) WHERE \(DataManager.episodeTableName).uuid IN \(DBUtils.valuesQuestionMarks(amount: uuids.count)) and
```